dvc/tree/azure.py

```diff
@@ -42,7 +42,7 @@ def __init__(self, repo, config):
 
         self._account_url = None
         if not self._conn_str:
-            name = self._az_config.get("storage", "account", None)
+            name = config.get("account") or self._az_config.get("storage", "account", None)
```

Comment:
Btw, I suppose you were testing out things through API? If so, for CLI we'll need to add `account` and `sas_token` to https://github.com/iterative/dvc/blob/67ddb7b41f8ba382048327fa5b5520168ab9c43f/dvc/config.py#L193

Reply:
Sorry, this should not have gone through.